codalab/rest/util.py

```diff
@@ -0,0 +1,65 @@
+"""
+Helper functions for working with the BundleModel.
+Most of these are adapted from the LocalBundleClient methods,
+Placed in this central location to prevent circular imports.
+"""
+import httplib
+import os
+import re
+
+from bottle import abort, local, request
+from marshmallow_jsonapi import Schema
```

Comment:
These are unused imports. Please don't leave around unused imports. I use Eclipse and that points these out. If you're not an Eclipse user, try pylint.